Developer-Guide.md

```diff
@@ -596,6 +567,16 @@ $ sudo awk '{if (/^\[proxy\.kata\]/) {got=1}; if (got == 1 && /^.*enable_debug/)
 $ sudo install -o root -g root -m 0640 /tmp/configuration.toml /etc/kata-containers/
 ```
 
+### Enable debug console for the agent
+
+Add `agent.debug_console` to the guest kernel command line to allow the agent process to start a debug console. 
+```
+$ conf="/etc/kata-containers/configuration.toml"
```

Comment:
I wonder if we should just say to add agent.debug_console instead, to avoid (1) globbing what may have been there and, (2) allowing to use the correct conf file (in case they aren't using /etc path).

Reply:
Agreed. Something like we specify for [enabling full debug](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#enable-full-debug):

```sh
sudo sed -i -e 's/^kernel_params = "\(.*\)"/kernel_params = "\1 agent.debug_console=true"/g' "$conf"
```

Reply:
@egernst @jodh-intel I have made some changes to the document. PTAL.